mp/src/game/client/c_baseanimating.cpp

```diff
@@ -3610,6 +3611,9 @@ void C_BaseAnimating::DoAnimationEvents( CStudioHdr *pStudioHdr )
 //-----------------------------------------------------------------------------
 bool C_BaseAnimating::DispatchMuzzleEffect( const char *options, bool isFirstPerson )
 {
+	if (!r_drawviewmodel.GetBool())
```

Comment:
Should this also check if `isFirstPerson`?

Reply:
Yeah, definitely, as "isFirstPerson" relates to whether or not it's the NPC or the player; fixing that right now